packages/tpetra/classic/NodeAPI/Kokkos_DefaultNode.hpp

```diff
@@ -62,40 +62,16 @@ namespace Details {
   /// \brief Create and return a Kokkos Node instance.
   /// \tparam NodeType The Kokkos Node type.
   ///
-  /// \warning This function is <i>not</i> safe to be called by
-  ///   multiple threads simultaneously.  The first call to this
-  ///   function must be serialized.  Also, RCP is not currently
-  ///   thread safe.
-  ///
-  /// \param params [in/out] On input: Any parameters that the Kokkos
-  ///   Node accepts.  On output, the list may be modified to include
-  ///   missing parameters and their default values.  If params is
-  ///   null, default parameters will be used.
-  ///
-  /// Every Kokkos Node's constructor takes a Teuchos::ParameterList.
-  /// We presume that for every Kokkos Node, if that list of
-  /// parameters is empty, then the Node will use default parameters.
-  /// This is true for all the Node types implemented in Kokkos.
+  /// This function ONLY exists for backwards compatibility.
```

Comment:
@kddevin Thanks for pointing this out!  Before doing that, I would like to change downstream code not to call this method.  Otherwise, Albany etc. will get a bazillion deprecation warnings.

Reply:
Since that would touch a lot of downstream code, I'd rather save it for a separate PR.